pkgs/misc/drivers/epson_201601w/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib, stdenv, fetchurl, rpmextract, autoreconfHook, file, libjpeg, cups }:
+
+let
+  version = "1.0.1";
+  filterversion = "1.0.2";
+in
+  stdenv.mkDerivation {
+
+    pname = "epson_201601w";
+    inherit version;
+
+    src = fetchurl {
+      # NOTE: Don't forget to update the webarchive link too!
+      urls = [
+        "https://download3.ebz.epson.net/dsc/f/03/00/15/66/51/1046e0a9f8d8ec892806a8d4921335cf6f5fd1ea/epson-inkjet-printer-201601w-${version}-1.src.rpm"
+        "https://web.archive.org/web/20240601191136/https://download3.ebz.epson.net/dsc/f/03/00/15/66/51/1046e0a9f8d8ec892806a8d4921335cf6f5fd1ea/epson-inkjet-printer-201601w-${version}-1.src.rpm"
+      ];
+      sha256 = "048d72dd4dc4bd5aaa15f43b6274318ae21bcba189e41d13082da3407d54a2cd";
+    };
+
+    nativeBuildInputs = [ rpmextract autoreconfHook file ];
+
+    buildInputs = [ libjpeg cups ];
+
+    unpackPhase = ''
+      rpmextract $src
+      tar -zxf epson-inkjet-printer-201601w-${version}.tar.gz
+      tar -zxf epson-inkjet-printer-filter-${filterversion}.tar.gz
+      for ppd in epson-inkjet-printer-201601w-${version}/ppds/*; do
+        substituteInPlace $ppd --replace "/opt/epson-inkjet-printer-201601w" "$out"
+        substituteInPlace $ppd --replace "/cups/lib" "/lib/cups"
+      done
+      cd epson-inkjet-printer-filter-${filterversion}
+    '';
+
+    preConfigure = ''
+      chmod +x configure
+      export LDFLAGS="$LDFLAGS -Wl,--no-as-needed"
+    '';
+
+    postInstall = ''
+      cd ../epson-inkjet-printer-201601w-${version}
+      cp -a lib64 resource watermark $out
+      mkdir -p $out/share/cups/model/epson-inkjet-printer-201601w
+      cp -a ppds $out/share/cups/model/epson-inkjet-printer-201601w/
+      cp -a Manual.txt $out/doc/
+      cp -a README $out/doc/README.driver
+    '';
+
+    meta = with lib; {
```

Comment:
`with lib;` is problematic antipattern. Use `with lib.licenses;` and `with lib.maintainers;` below
See https://github.com/NixOS/nixpkgs/issues/208242